README.md

```diff
@@ -59,26 +86,30 @@ LongCounter counter = meter
 ```
 
-A full setup is provided in our [example project](example/).
+A full setup is provided in our [example project](example/src/main/java/com/dynatrace/opentelemetry/metric/example/DynatraceExporterExample.java)
 
 ### Configuration
 
-The exporter allows for configuring the following settings using its builder:
+The exporter allows for configuring the following settings using its builder (`DynatraceMetricExporter.builder()`):
 
 #### Dynatrace API Endpoint
 
-The endpoint to which the metrics are sent is specified using `setUrl`.
+If a OneAgent is installed on the host, it can provide a local endpoint for ingesting metrics without the need for an API token.
+The [OneAgent metric API documentation](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/) provides information on how to set up a local OneAgent endpoint.
+Using the local API endpoint, the host ID and host name context are automatically added to each metric as dimensions.
 
-Given an environment ID `myenv123` on Dynatrace SaaS, the [metrics ingest endpoint](https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/post-ingest-metrics/) would be `https://myenv123.live.dynatrace.com/api/v2/metrics/ingest`.
+If no OneAgent is running on the host or if metrics should be sent to a different endpoint, the `setUrl` method allows for setting that endpoint.
 
-If a OneAgent is installed on the host, it can provide a local endpoint for providing metrics directly without the need for an API token.
-This feature is currently in an Early Adopter phase and has to be enabled as described in the [OneAgent metric API documentation](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/).
-Using the local API endpoint, the host ID and host name context are automatically added to each metric as dimensions.
-The default metric API endpoint exposed by the OneAgent is `http://localhost:14499/metrics/ingest`.
+The metrics ingest endpoint URL looks like:
+
+- `https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest` on SaaS deployments.
+- `https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest` on managed deployments.
 
 #### Dynatrace API Token
 
-The Dynatrace API token to be used by the exporter is specified using `setApiToken` and could, for example, be read from an environment variable.
+The Dynatrace API token to be used by the exporter is specified using `setApiToken`.
+The token could, for example, be read from an environment variable.
+It should not be hardcoded into the code, especially if that code is stored in a VCS.
 
 Creating an API token for your Dynatrace environment is described in the [Dynatrace API documentation](https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/).
 The scope required for sending metrics is the `Ingest metrics` scope in the **API v2** section:
```

Comment:
```suggestion
The permission required for sending metrics is the `Ingest metrics` (`metrics.ingest`) permission in the **API v2** section
and it is recommended to limit scope to only this permission:
```

Reply:
```suggestion
The permission required for sending metrics is the `Ingest metrics` (`metrics.ingest`) permission in the **API v2** section and it is recommended to limit scope to only this permission:
```
Keep it on one line, just nit-picking here.